src/sentry/mediators/sentry_app_components/preparer.py

```diff
@@ -54,7 +54,11 @@ def _prepare_field(self, field):
             field.update({'choices': field['options']})
 
         if 'uri' in field:
-            field.update(self._request(field['uri']))
+            if 'async' in field:
+                # react-select expects a 'url' field
+                field.update({'url': field['uri']})
```

Comment:
Does this need to be expanded out to a full URL (opposed to just the URI portion?)

Reply:
we don't need to generate the full url here cause we do that in the `SelectRequester`, I think we just need the url field because react-select expects that to be true if it's async (I think)